server/server.js

```diff
@@ -2,6 +2,7 @@ import mongoose from "mongoose";
 import makeApp from "./index.js";
 
 const app = makeApp(mongoose); // connect to databse
+// const app = makeApp(); // start without connecting to database for testing
```

Comment:
Should remove comments like this, we want to keep the code base fairly clean from debug comments.